### ISSUE TITLE
Is app still maintained?

### ISSUE THREAD

**Issue by TheMatten** [aricneto/TwistyTimer#125]
It seems like there's still interest in the app, but last commit was made in 2017 and from then new issues did not get any reaction. Is this only some sort of "pause" from development or should we consider creating fork?

**Comment by aricneto** [comment]
Hey!
I've unfortunately been very away from my personal projects this year. Turns out studying Computer Engineering takes way more of my personal time than I'd have imagined. I _really_ want to resume development of this app, and I have a lot of ideas and layouts planned.

I'd consider it more of a pause. Hopefully not a very long one, I'm trying to be more organized and create a schedule to be able to keep my projects going.

Anyway, it'll resume when I'm confident I can manage and maintain it. I don't want to start development and then stop it again like I did before. In the meantime (which I hope won't be long), feel free to fork it, I'd love to see what people come up with, and I could integrate the changes into the app when I'm back.

Thanks for your interest in the app!
